### DISCUSSION TITLE
HimangshuSharma Progress

### DISCUSSION THREAD

**Discussion by HimangshuSharma01** [sanjay-kv/Open-source-Practice#1151] [General]
![image](https://github.com/user-attachments/assets/225bf692-3c11-4523-a73b-f5224899cafa)
@sanjay-kv kindly give me the comment

**Comment by sanjay-kv** [comment]
Hi, Thanks for contributing to this repo.
If you would like to contribute more and learn. Join my github org https://github.com/recodehive

Playlist on YouTube Watch to learn more about how you can earn badge and free GitHub Pro. 
[GitHub Tutorials](https://stream.recodehive.com/github)

Now if you want to have a free github profile review: Check this out: https://topmate.io/sanjaykv/1135910/pay
Thanks, 
behalf of (Sanjay Viswanathan)

**Reply by sanjay-kv** [reply]
@HS202022 

here is the comment, we can automate this comment. 
let me know if you find any article related to it so we can share and learn each other

**Reply by sanjay-kv** [reply]
@HS202022 

your profile review:
https://www.loom.com/share/26aebe0a6a8242aeb27ecf1d98351df7?sid=e0d4658c-394b-4469-8159-8b8bde6b2ca4


Module 4: How to get Github Achievments badges
10.How to get github achievements: https://youtu.be/sbyXpflAXkQ
11. How to get Pull shark badge on GitHub: https://youtu.be/7uKMWBFN2jQ
12. How to get Galaxy Brain badge on GitHub: https://youtu.be/v2Pai1TY_Lg
13. How to get Starstruck badge on GitHub: https://youtu.be/P-P3L7YzlyE
14. How to get Quick Draw badge on GitHub: https://youtu.be/BNKSlT8jLQ0
15. How to get Yolo badge on GitHub: https://youtu.be/GnHNScuGKrg
16. How to get pair on GitHub: https://youtu.be/ZoNO_e8PjiM


watch all these videos

Module 5: How to activate sponsership on GitHub
17. How to Sponsor developers on GitHub: https://youtu.be/krgKPvWNPLQ
18. How to get GitHub Sponser: https://youtu.be/RSR5E1bhu5Y
19. How to setup GitHub sponser: https://youtu.be/z579SrLdk2I

**Reply by HimangshuSharma01** [reply]
i activated the sponsorship and updated the profile i will change the profile picture very soon

**Reply by sanjay-kv** [reply]
https://github.com/sponsors/eddiejaoude

make it something like this

**Reply by HimangshuSharma01** [reply]
> https://github.com/sponsors/eddiejaoude
> 
> make it something like this

i made it

**Comment by sanjay-kv** [comment]
![image](https://github.com/user-attachments/assets/90c1bcb5-9b94-43da-a0d4-8f74c2f53ae8)
@HimangshuSharma01 your graph is improving

**Comment by sanjay-kv** [comment]
Organisation to join:
AccessibleForAll: https://github.com/AccessibleForAll
Devs-Dungeon: https://github.com/Devs-Dungeon
Magic-Academy: https://github.com/Magic-Academy
EddieHubCommunity: https://github.com/EddieHubCommunity
Bauddhik-Geeks: https://github.com/Bauddhik-Geeks
github-craftwork: https://github.com/github-craftwork

Signin up for developer program member: https://github.com/settings/profile#github-developer-program

apply for students developer pack: https://education.github.com/pack

@HimangshuSharma01 do join these org @HimangshuSharma01

**Comment by sanjay-kv** [comment]
![image](https://github.com/user-attachments/assets/73f9faab-e091-47e4-a82b-f6019ac31b48)


highlight only 2 projects

**Comment by sanjay-kv** [comment]
@HimangshuSharma01 Change min sponsership $1

**Reply by sanjay-kv** [reply]
![image](https://github.com/user-attachments/assets/70eff30f-3b19-49a2-85bb-6f99ca6fb1f4)

**Reply by HimangshuSharma01** [reply]
> @HimangshuSharma01 Change min sponsership $1

@sanjay-kv I did it sir

**Comment by sanjay-kv** [comment]
your current profile: im just keeping ss so u can come back later and see what all changed
![screencapture-github-HimangshuSharma01-2024-10-17-14_42_00](https://github.com/user-attachments/assets/de925c0a-a022-4cbd-8864-4c61b2d0e4fe)

**Comment by sanjay-kv** [comment]
![image](https://github.com/user-attachments/assets/30a07659-2aac-4349-8d3f-bf53561cfac9)


add labels to these merged once.

**Comment by HimangshuSharma01** [comment]
> ![image](https://github.com/user-attachments/assets/30a07659-2aac-4349-8d3f-bf53561cfac9)
> 
> 
> add labels to these merged once. 

Ok sir
